packages/sit-onyx/src/styles/grid.scss

```
@@ -40,14 +40,43 @@ $sidebar-breakpoints: (
     gap: var(--onyx-grid-gutter);
   }
 
+  /**
+  * Grid container classes for "xl" and "lg" breakpoint variants with 16 or 20 columns
+  */
+  @include container(min, lg) {
+    .onyx-grid.onyx-grid-xl-20,
+    .onyx-grid-layout.onyx-grid-xl-20,
+    .onyx-grid-container.onyx-grid-xl-20,
+    .onyx-grid.onyx-grid-lg-16,
+    .onyx-grid-layout.onyx-grid-lg-16,
+    .onyx-grid-container.onyx-grid-lg-16 {
+      --onyx-grid-columns: 16;
+    }
+  }
+
+  @include container(min, xl) {
+    .onyx-grid.onyx-grid-xl-20,
+    .onyx-grid-layout.onyx-grid-xl-20,
+    .onyx-grid-container.onyx-grid-xl-20 {
+      --onyx-grid-columns: 20;
+    }
+
+    .onyx-grid.onyx-grid-xl-16,
+    .onyx-grid-layout.onyx-grid-xl-16,
+    .onyx-grid-container.onyx-grid-xl-16 {
+      --onyx-grid-columns: 16;
+    }
+  }
```

```suggestion
  @include container(min, lg) {
    .onyx-grid,
    .onyx-grid-layout,
    .onyx-grid-container {
      &.onyx-grid-xl-20,
      &.onyx-grid-lg-16 {
        --onyx-grid-columns: 16;
      }
    }
  }

  @include container(min, xl) {
    .onyx-grid,
    .onyx-grid-layout,
    .onyx-grid-container {
      &.onyx-grid-xl-20 {
        --onyx-grid-columns: 20;
      }
    }

    .onyx-grid,
    .onyx-grid-layout,
    .onyx-grid-container {
      &.onyx-grid-xl-16 {
        --onyx-grid-columns: 16;
      }
    }
  }
```

This should do the same + is a little better to read in my opinion :)